release.sh

```diff
@@ -2469,7 +2469,7 @@ if [ -z "$skip_zipfile" ]; then
 	fi
 
 	if [ -n "$GITHUB_ACTIONS" ]; then
-		echo "::set-output name=archive_path::${archive}"
+		echo "archive_path=${archive}" >> $GITHUB_OUTPUT
```

Comment:
Use `ARCHIVE_PATH=${archive}`

Reply:
Considering the findings of @Molkree below, this shouldn't matter, but I'll change it as requested.